web_widget_timepicker/README.rst

```diff
@@ -24,22 +23,21 @@ Usage
 =====
 
 This module defines a new widget type for form views input fileds.
-
 Set the attribute ``widget=timepicker`` in a ``field`` tag in a form view.
 
-You can pass all options through the "timepicker" field in the options::
+You can pass custom options through the "timepicker" field in the options attribute:
 
     ...
-    <field name="mytimefieldname" `widget=timepicker`` data-options="{'step': '30', 'disableTextInput': false}"/>
+    <field name="mytimefieldname" `widget=timepicker`` options="{'step': '30', 'disableTextInput': false}"/>
     ...
 
-See the available options at https://github.com/jonthornton/jquery-timepicker#timepicker-plugin-for-jquery
+See the available options at https://github.com/jonthornton/jquery-timepicker#timepicker-plugin-for-jquery.
 
 
-ToDo
-====
+Known issues / Roadmap
+======================
 
-Sanity check on options available in field defintion as override options for timepicker widget.
+* Absolutely no sanity check or validation on options.
```

Comment:
I think this might confuse users, because it documents the development process, not the outcome